.travis.yml

```diff
@@ -29,9 +29,9 @@ install:
   # -----------------
   - export CONDA_BASE=http://repo.continuum.io/miniconda/Miniconda
```

Comment:
It auto-redirects to the https version, which wget seems OK with... Best move to https anyway though.